src/views/SignupView/StudentForm.vue

```diff
@@ -488,36 +518,8 @@ export default {
           }
         });
     },
-    submitWaitlist() {
-      // reset error msg from server
-      this.msg = "";
-
-      // validate input
-      this.errors = [];
-      this.invalidInputs = [];
-
-      if (!this.waitlist.email) {
-        this.errors.push("An email address is required.");
-      } else if (!validator.isEmail(this.waitlist.email)) {
-        this.errors.push(
-          this.waitlist.email + " is not a valid email address."
-        );
-      }
-      if (this.errors.length) {
-        return;
-      }
-
-      NetworkService.joinSchoolApprovalWaitlist(this, {
-        email: this.waitlist.email,
-        schoolUpchieveId: this.eligibility.highSchool.upchieveId
-      })
-        .then(() => {
-          this.step = "step-1-waitlist-success";
-          this.$router.push("/sign-up/student/waitlist-success");
-        })
-        .catch(err => {
-          this.msg = err.message;
-        });
+    ineligibleContinue() {
+      this.$router.push("https://upchieve.org/parents");
```

Comment:
Nice work!! It looks great

This part here took me to:
<img width="553" alt="parenturl" src="https://user-images.githubusercontent.com/35900194/87834050-66c33300-c857-11ea-822a-0791a58278b9.png">

Reply:
Oo yeah good catch